mods/ra/rules/vehicles.yaml

```diff
@@ -792,6 +792,13 @@ CTNK:
 		Prerequisites: vehicles.upgraded
 	Selectable:
 		DecorationBounds: 1280, 1280
+	Contrail@Teleport: ##HACK: add a trace for teleport. 
```

Comment:
Remove the `##HACK: add a trace for teleport.` as `Contrail@Teleport` already says what it does.

Reply:
ok